src/fileformats/ocd_file_import.cpp

```diff
@@ -800,6 +802,12 @@ void OcdFileImport::importSymbols(const OcdFile< F >& file)
 		symbol_index[ocd_symbol.number] = symbol;
 	}
 	resolveSubsymbols();
+	
+	for (auto ocd_symbol_entry : file.symbols())
+	{
+		auto& ocd_symbol = *ocd_symbol_entry.entity;
+		dropRedundantIcon(symbol_index[ocd_symbol.number], ocd_symbol);
```

Comment:
Error case needs to be handled here.

If symbol import fails above, the corresponding `symbol_index` entry is `nullptr`. `dropRedundantIcon()` does not handle the `symbol == nullptr` case well.